cmd/server/cadence/server.go

```diff
@@ -220,7 +223,20 @@ func (s *server) startService() common.Daemon {
 			log.Fatalf("not able to find advanced visibility store in config: %v", advancedVisStoreKey)
 		}
 
-		params.ESConfig = advancedVisStore.ElasticSearch
+		if params.PersistenceConfig.AdvancedVisibilityStore == "pinot-visibility" {
```

Comment:
use `PinotVisibilityAppName`

Reply:
I thought the plugins (like what mySQL/Cassandra/mongoDB/postgre have) were not for visibility stores? Because we don't have the plugin files for ElasticSearch, and we don't need to register ElasticSearch as a plugin.

Reply:
You are right, it is better to use constants here or it can mess up since it is used for multiple places. Updated.